autospec/config.py

```diff
@@ -711,11 +711,8 @@ def write_default_conf_file(name, description):
 
     content = read_conf_file(os.path.join(path, "attrs"))
     for line in content:
-        attr = re.split(r'\(|\)|,', line)
-        attr = [a.strip() for a in attr]
-        filename = attr.pop()
         print("attr for: %s." % filename)
-        filemanager.attrs[filename] = attr
+    filemanager.attrs += content
```

Comment:
So I'm inclined to just delete this entirely as it isn't used, it is pretty old code and more headache than value when looking at this change.

Reply:
I'll split the removal off to a new PR and rebase.

Reply:
Thanks!